src/main/daml/Daml/Finance/Interface/Util/Disclosure.daml

```diff
@@ -85,3 +86,30 @@ interface Disclosure where
 -- ```
 flattenObservers : PartiesMap -> Parties
 flattenObservers = mconcat . M.values
+
+-- | Disclose any template instance implementing the disclosure interface.
```

Comment:
```suggestion
-- | Disclose a template instance implementing the disclosure interface.
```

I think the "any" is slightly confusing in this case, as one might think it applies to more than one contrct

Reply:
done